### ISSUE TITLE
Figure 5, decide number of common names to show

### ISSUE THREAD

**Issue by odontaster** [slarge/fisheryO#50]
The group "other" is large, but is formed by small amounts of many species, so when adding more names, they just pack down in the graph, messing the names quite a bit. I think that hake, norway lobster, whiting and haddock are those more visible. 
![cs_figure5](https://user-images.githubusercontent.com/18613829/45357046-6dcaeb80-b5c5-11e8-85ac-9fd1f2e1e789.png)
![cs_figure5v 2](https://user-images.githubusercontent.com/18613829/45357051-6efc1880-b5c5-11e8-8da3-130672bce0c9.png)

**Comment by clordan** [comment]
Hi I think this is better, you could also show Anglerfish and possibly megrim which should be quite large..... larger than cod at least.

**Comment by sarahlmillar** [comment]
I agree the fewer species is better, it makes no sense to have them all on there and not see anything. We can explain it in the text. however - if anglerfish and cod are large they should already be showing on there. We should check total anglerfish landings....

**Comment by odontaster** [comment]
Thank both. I found out that catches after 2006 were not showing up, sorry. Now they do.
Still with 12 labels Anglerfish and megrim don't appear because the order is set by the sum of catches of the whole time-series, so still cod is larger. If you want I can try to select those two and see how it looks. 
Let me know!
![cs_figure5](https://user-images.githubusercontent.com/18613829/46399360-c179b200-c6f7-11e8-929b-394de93aba0e.png)

**Comment by sarahlmillar** [comment]
could you try it with those 2 (anglerfish and hake) selected in as well and let us see it? That would be great

**Comment by odontaster** [comment]
Ok,
I merged catches this way:
Anglerfishes nei= Anglerfishes nei + Angler(=Monk) + Monfishes nei
Megrims nei = Megrims nei + Megrim

Like this, Anglerfishes nei ranks up right after European hake, but Megrims nei is still very low in the rank, so I forced it to show up (although it is ranked 17th).
Let me know, cheers
![cs_figure5 anglermegrim](https://user-images.githubusercontent.com/18613829/46734497-b0024e00-cc93-11e8-927c-91c5a991ef91.png)

**Comment by sarahlmillar** [comment]
For me, we shouldn't force it in there if its not in the top list unless we are specifically referring to it in the text (for instance - megrim is a high value species etc etc). But @Colm to decide....

**Comment by sarahlmillar** [comment]
monkfish/angelerfish is good
